auth/user.go

```diff
@@ -135,8 +134,20 @@ func GetCurrentUserFromDB(req *http.Request) (*User, error) {
 	return nil, errors.New("error fetching user from db")
 }
 
+func newDroneClient(apiToken string) drone.Client {
+	droneURL := viper.GetString("drone.url")
```

Comment:
please also add this parameter to the config.toml.example

Reply:
Added, thanks!